lumigator/schemas/lumigator_schemas/workflows.py

```diff
@@ -28,6 +28,7 @@ class WorkflowCreateRequest(BaseModel):
     system_prompt: str | None = None
     inference_output_field: str = "predictions"
     config_template: str | None = None
+    job_timeout: NonNegativeInt = 60 * 10
```

Comment:
this parameter needs to have a unit given it. Maybe `job_timeout_sec`?

Reply:
LGTM